packages/design-system/src/components/Form/Primitives/Primitives.stories.mdx

```diff
@@ -0,0 +1,54 @@
+import { ArgsTable, Meta, Canvas, Story } from '@storybook/addon-docs';
+// import { FigmaImage, Use } from '~docs';
+
+import InlineMessage from '../../InlineMessage/InlineMessage';
+import * as Stories from './Primitives.stories';
+import { StackVertical } from '../../Stack';
+
+<Meta
+	title="[WIP] Components/Form/Primitives (unexported)"
+	parameters={{
+		status: { figma: 'wip', storybook: 'wip', react: 'wip', i18n: 'wip' },
+		figmaLink: 'https://www.figma.com/file/6XBBCBVOhyX6yTaNcZQiti/Empty-States',
+	}}
+/>
+
+# Form Primitives
+
+<StackVertical gap="0" padding={{ x: '0', y: 'M' }}>
+	<InlineMessage.Information
+		withBackground
+		description="Primitives are unexported components used within
+    the design system. They are composed together to create the public components."
+	/>
+</StackVertical>
+
+## Textarea
+
+<Canvas>
+	<Story story={Stories.TextareaPrimitive} />
+</Canvas>
+
+## Input
+
+<Canvas>
+	<Story story={Stories.InputPrimitive} />
```

Comment:
Affix is not aligned

<img width="939" alt="Screenshot 2022-05-16 at 11 45 35" src="https://user-images.githubusercontent.com/58977230/168565759-808ccc68-8dcc-4143-9433-516ba30136e2.png">

Reply:
Yup :) Affix work will be a separate PR, but thanks!